Makefile

```diff
@@ -25,14 +25,27 @@ GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 all: binary
 
-binary:
+binary: ${DEST}skopeo
+
+# Build a docker image (skopeobuild) that has everything we need to build.
+# Then do the build and the output (skopeo) shold appear in ${DEST} dir (or .).
+${DEST}skopeo: cmd/skopeo
+	docker build ${DOCKER_BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
@mtrmac correct, once built this step in the Makefile should almost be a no-op because docker will use the cache for each step in `Dockefile.build`.